Modify Design Cards

filippoweb3 opened the pull request:
Current Cards do not adapt to screen size

- Cards
- Banner

Commit:
modified cards

Commit:
cards for videos

Commit:
card size

Commit:
added badges

Commit:
modified badges

Commit:
minor edits badges

Commit:
edited video page

Commit:
edit vault page

Commit:
edit dashboard page

Commit:
edit delegation dashboard

Commit:
edit NFTs

Commit:
edit scams

Commit:
edit opengov

Commit:
edit DOT page

Commit:
edits account guides

Commit:
edit bounty guides

Commit:
final edits

DrW3RK commented:
If there is an accessibility issue with the Wiki, we need to sort it out with the web design team. 

<img width="771" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/3ea6a25e-8d62-4ad0-819a-4631ff6d78cd">

The elements here are redundant. The "click" text, as well as "videos youtube click" can be removed, which keeps it concise. Also, "built with love" with Polkadot does not go with the formal and technical tone of the Wiki.

The same goes with the redundancy with the the hyperlinks on the image cards and the numerous "view page" hyperlinks is visually unappealing. Why have you chosen to make these changes?

<img width="855" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/51fb544f-b95e-4c5d-9e56-70d6f9241369">

Commit:
modified cards + removed badges

filippoweb3 commented:
@DrW3RK I removed the badges and modified the cards (see below)

![Screenshot 2024-01-16 at 16 20 45](https://github.com/w3f/polkadot-wiki/assets/110459737/c7ded777-29ee-44b9-a962-037242ec23e9)

The title of the cards has the link. If you are ok with it I can transfer those changes to all other pages that are affected.

profradha commented on docs/general/governance-apps.md:
```diff
@@ -32,7 +32,18 @@ To make OpenGov multi-role delegation easy and intuitive,
+    <p>
+      Delegation Dashboard Tutorial
+      <br />
+      <a href="https://www.youtube.com/watch?v=RapBYZc5ZPo">» view video</a>
```
```suggestion
      <a href="https://www.youtube.com/watch?v=RapBYZc5ZPo">Delegation Dashboard Tutorial</a>
```

profradha commented on docs/general/polkadot-vault.md:
```diff
@@ -32,7 +32,18 @@ Vault app.
+    <p>
+      How to Use Polkadot Vault
+      <br />
+      <a href="https://www.youtube.com/watch?v=IG_RGLsb2g0">» view video</a>
```
```suggestion
      <a href="https://www.youtube.com/watch?v=IG_RGLsb2g0"> How to Use Polkadot Vault</a>
```

profradha commented on docs/general/scams.md:
```diff
@@ -19,7 +19,18 @@ with anyone, they can access your account, including your funds. This informatio
+    <p>
+      Stay Safe in Polkadot
+      <br />
+      <a href="https://www.youtube.com/watch?v=ARsdXZycJAg">» view video</a>
```
```suggestion
      <a href="https://www.youtube.com/watch?v=ARsdXZycJAg"> Stay Safe in Polkadot</a>
```

profradha commented on docs/general/staking-apps.md:
```diff
@@ -39,54 +39,61 @@ You can find more information about the Polkadot staking dashboard on the dedica
+    <p>
+      Nominate
+      <br />
+      <a href="https://youtu.be/F59N3YKYCRs?feature=shared">» view video</a>
```
```suggestion
      <a href="https://youtu.be/F59N3YKYCRs?feature=shared">Stake your DOT</a>
```

DrW3RK reviewed (approved):
Looks good. Just modify the links like below and the PR is good to go. Thank you.

filippoweb3 committed:
Update docs/general/governance-apps.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/scams.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/staking-apps.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-vault.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
edits to all cards
